README.md

```diff
@@ -1,11 +1,21 @@
-# kokkos-core-documentation-website
+# kokkos-core-documentation project
 Welcome to the Kokkos Documentation repository.  This is the source for https://kokkos.github.io/kokkos-core-wiki/.
 
 ## Requirements to build html page locally
 
-This is needed just for local render of documentation, so it can be checked before push.
-Requirements are in `build_requirements.txt`
-Could be installed with: `pip install -r build_requirements.txt`
+The library-level dependencies needed for Kokkos documentation are:
+
+- Sphinx
+- furo
+- myst-parser
+- sphinx-copybutton
+- m2r2
```

Comment:
i think all this above is superfluous, and we are just duplicating what is writte in the `build_requirements.txt`.

Reply:
Agreed, please refer to the requirement file